pl_bolts/models/rl/common/experience.py

```diff
@@ -29,8 +29,10 @@ class RLDataset(IterableDataset):
     """
 
     def __init__(self, buffer: Buffer, sample_size: int = 1) -> None:
-        warnings.warn("Deprecated, these functions have been moved to pl_bolts.datamodules.experience_source.py",
-                      DeprecationWarning)
+        warnings.warn(
```

Comment:
add info when it will be removed

Reply:
when should it be removed?

Reply:
as there is not any release yet I would drop this message completely

Reply:
I have removed this as It shouldn't be used going forward and could cause compatibility issues if left in